SPECS/cloud-hypervisor/cloud-hypervisor.spec

```diff
@@ -75,6 +80,12 @@ tar xf %{SOURCE1}
 pushd vendor/libz-sys/src/zlib
 %patch0 -p1
 popd
+pushd vendor/vmm-sys-util
+%patch1 -p1
+popd
+%patch2 -p1	
+%patch3 -p1
+%patch4 -p1
```

Comment:
Could we just bundle this into one patch and also update the paths in `Patch1`, so we can drop the pushd-pop stuff?

Reply:
Got it. Please just pick the fast-track version while you resolve the merge conflicts in the cherry-pick PR once this is one is merged.